### DISCUSSION TITLE
Installing on Le Potato and not prompted to setup admin user or default instance

### DISCUSSION THREAD

**Discussion by tekjock** [paukstelis/octoprint_deploy#100] [General]
When trying to setup the default instance it does not seem to be setting up the default instance, nor prompting me to 'Setup your admin account'

It asks about haproxy and then asks which  video streamer, then it goes right into asking 'It is time to create your first OctoPrint instance!!!' it does not seem to be creating the default profile. I installed this on a Dell PC with Ubuntu and it worked.

**Comment by paukstelis** [comment]
The order has changed in the latest version (compared to the guide you are likely following). Just keep following the instructions in the script and you will get there.
